tests/test_geospatialscenario.py

```diff
@@ -184,16 +193,18 @@ def test_downselect_CONUS(monkeypatch):
         data=[
             [-99, -99, -1, "+100", "United States", "Alaska", "filler", 2],
             [-99, -99, -1, "+100", "United States", "Hawaii", "filler", 2],
+            [-99, -99, -1, "+100", "United States", "Alaska", "filler", 2],
+            [-99, -99, -1, "+100", "United States", "Hawaii", "filler", 2],
```

Comment:
I think the duplicate rows (and trailing comma?) should be deleted
```suggestion
        data=[
            [-99, -99, -1, "+100", "United States", "Alaska", "filler", 2],
            [-99, -99, -1, "+100", "United States", "Hawaii", "filler", 2]
```